.php-cs-fixer.dist.php

```diff
@@ -49,6 +49,40 @@
         'random_api_migration' => true,
         // 'static_lambda' => true, // @todo @keradus to fix last issues
     ])
+    ->setRuleCustomisationPolicy(new class implements PhpCsFixer\Config\RuleCustomisationPolicyInterface {
```

Comment:
that's cool, can you split that in a new PR on 7.4? this way we won't remove the attributes by mistake there either

Reply:
> that's cool

👍🏻 
would you prefer standalone file for named policy class? where shall I put it?

> 7.4

👍🏻